kpatch-build/create-diff-object.c

```diff
@@ -1063,12 +1117,7 @@ static void kpatch_correlate_static_local_variables(struct kpatch_elf *base,
 				ERROR("sections %s and %s aren't correlated for symbol %s",
 				      sym->sec->name, patched_sym->sec->name, sym->name);
 
-			log_debug("renaming and correlating static local %s to %s\n",
-				  patched_sym->name, sym->name);
-
-			patched_sym->name = strdup(sym->name);
-			sym->twin = patched_sym;
-			patched_sym->twin = sym;
+			CORRELATE_ELEMENTS(sym, patched_sym, "static local");
 
 			/* set initial status, might change */
 			sym->status = patched_sym->status = SAME;
```

Comment:
I think this is no longer needed since CORRELATE_ELEMENTS does this?

Reply:
Yes, well spotted!